assets/app/views/create.html

```diff
@@ -38,7 +38,7 @@
           <div class="row">
             <div class="col-md-12">
               <div class="separator"></div>
-              <h1>Create Using a Template</h1>
```

Comment:
again, you're not adding a template, you're adding things from a template... I was ok with the old header